src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -428,6 +433,8 @@ private static void createBranches(List<LfBus> lfBuses, LfNetwork lfNetwork, LfT
             LfBus lfBus2 = getLfBus(branch.getTerminal2(), lfNetwork, parameters.isBreakers());
             LfBranchImpl lfBranch = LfBranchImpl.create(branch, lfNetwork, lfBus1, lfBus2, topoConfig, parameters);
             addBranch(lfNetwork, lfBranch, report);
+            updateControlAreaBoundaryP(branch.getTerminal1(), loadingContext, lfBranch::getP1);
+            updateControlAreaBoundaryP(branch.getTerminal2(), loadingContext, lfBranch::getP2);
```

Comment:
`update` is not the right term, `add` instead ?

Reply:
changed this, mostly in 097d801814619091d7d66cafc06b7cca1e399e97 and f3930a88979fe7d13f1eba90fcec07554932f868